src/recipeDetail/Header.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledHeader = styled.div`
+  background-image: url(${props => props.url});
+  background-size: 100%;
+  height: 231px;
+  background-position: center center;
+  background-repeat: no-repeat;
+  margin-right: 0;
+  margin-left: 0;
+  margin-top: 30px;
+`;
+
+const StyledTitleContainer = styled.div`
+  background-color: #3d9970;
+  color: white;
+  height: 42px;
+  position: relative;
+  top: -10px;
+  margin: 0;
+  padding-left: 15px;
+  display: flex;
+  align-items: center;
+`;
+
+const StyledTitle = styled.h3`
+  color: white;
+`;
+
+export default function Header({ recipe }) {
+  return (
+    <StyledHeader url={recipe.image}>
+      <StyledTitleContainer>
```

Comment:
Is this header Container necessary? Maybe you can also style the h3 element accordingly